<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-077G3EM" ignored="0" name="RingPay" tests="3" failures="1" timestamp="2022-10-29T20:25:45 IST" time="745.162" errors="0">
  <testcase name="User_Playstore_Flow" time="126.064" classname="com.RingPay.TestScripts.RingPayTestScripts"/>
  <testcase name="userRegistrationFlow" time="552.242" classname="com.RingPay.TestScripts.RingPayTestScripts"/>
  <testcase name="merchantFlow" time="24.724" classname="com.RingPay.TestScripts.RingPayTestScripts">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Enter valid First Name ] but found [Enter valid First Name],
	expected [Enter valid Last Name ] but found [Enter valid Last Name],
	expected [ Check your connection &amp;amp; try again ] but found [Check your connection &amp;amp; try again]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Enter valid First Name ] but found [Enter valid First Name],
expected [Enter valid Last Name ] but found [Enter valid Last Name],
expected [ Check your connection & try again ] but found [Check your connection & try again]
at com.business.RingPay.RingPayBusinessLogic.merchantFlow(RingPayBusinessLogic.java:659)
at com.RingPay.TestScripts.RingPayTestScripts.merchantFlow(RingPayTestScripts.java:52)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- merchantFlow -->
</testsuite> <!-- RingPay -->
